src/components/navbar/navbar-container.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { RoutingNavbar } from './navbar';
+import { logout } from 'redux/modules/user';
+
+const mapDispatchToProps = dispatch => ({
+  logout: () => dispatch(logout())
+});
+
+export const NavbarContainer =
+  connect(
+    null,
+    mapDispatchToProps,
+  )(RoutingNavbar);
```

Comment:
I'm not sure if logout should be in the left side navbar. it should probably be in the header or in more likely in the right sidebar component. but we can keep it here in the meantime.

Reply:
Put this in the same file as the navbar, and reverse the order, so that your connected container component is put in withRouter, keep the name RoutingNavbar. 
export const RoutingNavbar = withRouter(NavbarContainer);